src/pudl/output/ferc1.py

```diff
@@ -47,6 +53,11 @@ def plants_utils_ferc1(pudl_engine):
         pandas.DataFrame: A DataFrame containing useful FERC Form 1 Plant and
         Utility information.
     """
+    logger.warning(
+        "pudl.output.ferc1.plants_utils_ferc1() will be deprecated in a future version of PUDL."
+        " In the future, call the PudlTabl.pu_ferc1() method or pull the plants_utils_ferc1 table"
+        "directly from the pudl.sqlite database."
+    )
```

Comment:
@zaneselvans should we just delete the python functions we are converting to views or should we add deprecation warnings?

Reply:
I think we'll need to hold off on deleting the old output table functions because they are used in downstream tables. For example, `pudl.output.ferc1.plants_utils_ferc1()` is still being called in `pudl.output.ferc1.plants_steam_ferc1()` which can't really depend on the `denorm_plants_utils_ferc1` asset until it is an asset itself.

Reply:
It's also helpful to keep the old functions around to make sure the conversion worked properly.